whotracksme/data/db.py

```diff
@@ -118,3 +124,194 @@ def create_tracker_map(db, with_iab_vendors=False):
 def get_iab_vendorlist():
     iab_list = json.loads(requests.get(iab_vendorlist_url).content)
     return iab_list
+
+
+INT_COLUMNS = ['site_reach_top10k', 'reach_rank', 'site_reach_rank', 'site_rank', 'tracker_rank']
+
+def get_column_type(col):
+    if col in INT_COLUMNS:
+        return 'INTEGER'
+    return 'REAL'
+
+BASE_DATA_COLUMNS = [
+    "cookies",
+    "bad_qs",
+    "tracked",
+    "https",
+    "requests",
+    "requests_tracking",
+    "content_length",
+    "requests_failed",
+    "has_blocking",
+    "script",
+    "iframe",
+    "beacon",
+    "image",
+    "stylesheet",
+    "font",
+    "xhr",
+    "plugin",
+    "media",
+    "referer_leaked",
+    "referer_leaked_header",
+    "referer_leaked_url",
+    "hosts",
+    "trackers",
+    "companies",
+]
+
+DATA_COLUMNS = {
+    'trackers': [
+        "reach",
+        "site_reach",
+        "site_reach_top10k",
+        "site_avg_frequency"
+    ] + BASE_DATA_COLUMNS + [
+        "reach_rank",
+        "site_reach_rank"
+    ],
+    'sites': [
+        'popularity'
+    ] + BASE_DATA_COLUMNS,
+    'sites_trackers': [
+        'site_proportion',
+        'tracker_proportion',
+        'site_rank',
+        'tracker_rank',
+    ] + BASE_DATA_COLUMNS
+}
+DATA_COLUMNS['companies'] = DATA_COLUMNS['trackers']
+DATA_COLUMNS['domains'] = DATA_COLUMNS['trackers']
+
+class WhoTracksMeDB:
```

Comment:
I think it would be nice  to have this class as a context manager, to be able to do things like:

```python
with WhoTracksMeDB() as db:
    ...
```

Which would also automatically close the connection afterwards.